src/Commands/Install/Publisher.php

```diff
@@ -58,12 +58,11 @@ public function publishClass(string $className): void
         $namespace    = explode('\\', $className);
         $rawClassName = array_pop($namespace);
         $namespace    = implode('\\', $namespace);
-        $newNamespace = 'Config\\' . $rawClassName;
 
         $content = file_get_contents($file);
 
         $replace = [
-            $namespace                           => $newNamespace,
+            $namespace                           => 'Config',
```

Comment:
Has this been tested? It seems like would be missing something and I'm pretty sure the publisher does its job correctly currently?

Reply:
This change is because of Fatal error: Cannot declare class Config\Auth\Auth, because the name is already in use in /app/Config/Auth.php on line 0
since after publish namespace becomes: namespace Config\Auth;